tests/Command/ConfigureTest.php

```diff
@@ -107,6 +103,7 @@ public function testShouldCreateConfigurationWithDifferentLocationOfFrameworkCon
             "\n" .
             "\n" .
             "\n" .
+            "\n" .
```

Comment:
Extra line of input needed for all tests with the new configure question included.